scripts/ci/rust_checks.py

```diff
@@ -62,111 +62,140 @@ def package_name_from_cargo_toml(cargo_toml_path: str) -> str:
 
 
 def main() -> None:
-    parser = argparse.ArgumentParser(description="Run Rust checks and tests")
+    checks = [
+        ("base_checks", base_checks),
+        ("sdk_variations", sdk_variations),
+        ("cargo_deny", cargo_deny),
+        ("wasm", wasm),
+        ("individual_examples", individual_examples),
+        ("individual_crates", individual_crates),
+        ("docs", docs),
+        ("tests", tests),
+    ]
+    check_names = [check[0] for check in checks]
+
+    parser = argparse.ArgumentParser(description="Run Rust checks and tests.")
     parser.add_argument(
-        "--skip-check-individual-crates",
-        help="If true, don't check individual crates in /crates/.",
-        action="store_true",
+        "--skip",
+        help="Skip all specified given checks but runs everything else.",
+        nargs="+",
+        type=str,
+        choices=check_names,
     )
     parser.add_argument(
-        "--skip-check-individual-examples",
-        help="If true, don't check individual examples in /examples/rust/.",
-        action="store_true",
+        "--only",
```

Comment:
The top of this file could use some example for common things one would like to do, e.g. `rust_checks.py --only wasm`